.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### All submissions:
+
+* [ ] Does your code follow
+  the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+
+<!-- Mark completed items with an [x] -->
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### Changes proposed in this Pull Request:
+
+<!-- Describe the changes made to this Pull Request and the reason for such changes. -->
+
+Closes # .
```

Comment:
Keen to move this up the top if others are ok with that (light preference - happy for it to stay here if that's more common).

Reply:
Updated in 83153f81e85c1ab3424c7618fd242ecd36e10673